### PR TITLE
change id to paketo-community/bundler

### PR DESCRIPTION
**Pull request by joshzarrabi**
- previously org.cloudfoundry.bundler

[Resolves paketo-community/bundler/issues/8]

### CODE REVIEW

File: buildpack.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 api = "0.2"
 
 [buildpack]
-  id = "org.cloudfoundry.bundler"
+  id = "paketo-community.bundler"
```
Comment (sclevine) [inline buildpack.toml:4]: Should this be `paketo-community/builder` instead?
Otherwise these buildpacks won't be publishable to the buildpack registry.

Comment (joshzarrabi) [reply buildpack.toml:4]: Yup, good catch. Thanks!

File: scripts/package.sh [hunk 2]
```diff
@@ -55,7 +55,7 @@ function main() {
         #use jam
         util::tools::jam::install --directory "${BUILDPACKDIR}/.bin"
         if [[ -z "${version:-}" ]]; then #version not provided, use latest git tag
-            git_tag=$(git describe --abbrev=0 --tags 2>/dev/null || echo "v0.0.0")
+            git_tag=$(git describe --tags `git rev-list --tags --max-count=1`)
```
Comment (arjun024) [inline scripts/package.sh:58]: Wouldn't this line error out when packaging a new cnb with no tags?

Comment (joshzarrabi) [reply scripts/package.sh:58]: yes, but its consistent with all of the other buildpacks

**Pull request merged**